src/components/FormInput.module.scss

```diff
@@ -0,0 +1,244 @@
+//form scss
+body {
+  width: 100%;
+  height: 100vh;
+  background: #EBF2FA;
+}
+
+.location {
+  margin-bottom: 8px;
+
+  &__discovery-icon {
+    stroke: #1b1c20;
+    margin: 0 7px;
+  }
+
+  &__form {
+    width: 100%;
+    background-color: white;
+    border: 1px solid #d5dfe9;
+    overflow: hidden;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    box-shadow: 0px 4px 4px #CBD5E1;
+    //size
+    border-radius: 13.7px;
+  }
+
+  &__title {
```

Comment:
we should prevent using !important, sometimes we can do it without using !important and sometimes not